content/articles/spotting-of-passing-objects-using-python/index.md

```diff
@@ -0,0 +1,165 @@
+﻿## Spotting of Passing Objects Using Python
+### Introduction
+Spotting of moving objects has become crucial in our lives today. Talking of security, cameras help us a great deal in executing this  
+function. Other key areas cut across tracking of living organisms,  vehicles in traffic control, and also replaceable where radar is discouraged.  
+We will be spotting objects in motion in this tutorial.  
+We will be requiring a few tools before carrying on, that is, the laptop's webcam and most of all an external library to be installed and downloaded which is the OpenCV module. With those at hand, we will be able to tackle our project. Let's  dive in.
+
+### Table of contents
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Creation of the spotting of passing objects](#creation-of-the-spotting-of-passing-objects)
+- [Results](#results)
+- [Conclusion](#conclusion) 
+
+
+
+### Prerequisites
+1. You should be an average-level programmer with basic coding skills in python.
+2. A brief overview of and prior knowledge to be equipped with before proceeding will be of great benefit. Following is the link directing you. [How to Adjust Image to Higher or Lower Resolution using Python](/engineering-education/how-to-adjust-image-to-higher-or-lower-resolution-using-python/)
+
+## Creation of the spotting of passing objects
+### Downloading and installing of external libraries
+### Step I:		
+First and foremost, we have to get **OpenCV** in place to be able to manipulate and work with our webcam. Also being a component used in the processing of pictures and live videos  giving solutions to computer vision rising issues.
+In your working environment, open the terminal and paste the command below to have the OpenCV downloaded and installed simultaneously.
+```bash
+ pip install OpenCV-Python
+```
+This library allows for modules such as cv2 to be called and used since cv2 will be playing a key role in our application.
+
+ ### Step II:
+ The second library to download and install will be **numpy**. Having interacted already with OpenCV and known its functions,  it takes the place of an image. So the numpy comes in since it is used in the expression of multi-dimensional arrays. For our case, our representation of the arrays will be the pixels. Likewise, as we had performed for the previous command, we will also be carrying it out same for numpy.
+ ```bash
+ pip install numpy
+ ```
+ ### Step III:
+ Lastly, we are going to install  **imutils** which is useful in  rotating, resizing, and skeletonizing images alongside OpenCV.
+ As you have been following, let's repeat the procedure in the above commands.
+ ```bash
+  pip install imutils
+ ```
+After successful installations, we will now jump into the code area.
+### Case 1:Importing the external libraries.
```

Comment:
You call something "Case" when there are different ways of doing the same thing.

Here, this is not relevant. Just have the topic name. Remove "Case 1".

Do the same for remaining